### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivan-shpynda**

### CODE REVIEW

**Review by Grygoriy-Shytikov — APPROVED** [review]
Думаю варто дещо поправити, а так Добре

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,26 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let sum = 0;
+  let isEven = 0;
```
Comment (varseniuk) [inline src/makeRobotAccountant.js:10]: Your `sum` and `isEven` are equal. Why do you need both of them? One is redundand

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -6,7 +6,26 @@
...
+
+  return function getSum(num1) {
+    // debugger;
```
[the remark this answers is not in the document]
Comment (varseniuk) [reply src/makeRobotAccountant.js:13]: Grygoriy is right

File: src/makeRobotAccountant.js [hunk 3]
```diff
@@ -6,7 +6,26 @@
...
+    isEven++;
+    sum++;
+
+    if (isEven > 3 && isEven % 2 === 0) {
```
Comment (varseniuk) [inline src/makeRobotAccountant.js:17]:
```suggestion
    if (sum > 3 && sum % 2 === 0) {
```
In this case you don't need `isEven`

**Review by varseniuk — CHANGES_REQUESTED** [review]
Let's remove some redundant code and it's done

**Review by varseniuk — APPROVED** [review]
Great job 🚀

**Pull request closed without merging**